doc/config.rst

```diff
@@ -295,6 +295,21 @@ General configuration
 
    .. versionadded:: 1.3
 
+.. confval:: manpages_url_pattern
+
+   A URL to cross-reference :rst:role:`manpage` directives. If this is
+   defined to ``https://manpages.debian.org/{manpage_full}``, the
+   :literal:`:manpage:`man(1)`` role will like to
+   <https://manpages.debian.org/man(1)>. The patterns available are:
+
+     * ``manpage_name`` - the manpage name (``man``)
+     * ``manpage_section`` - the manpage section (``1``)
+     * ``manpage_full`` - the whole string (``man(1)``)
+
+   This also supports manpages specified as ``man.1``.
+
+   .. versionadded:: 1.7
```

Comment:
Please mention to this confval at an introduction of :manpage: role.
http://www.sphinx-doc.org/en/stable/markup/inline.html#role-manpage

Reply:
done.

Reply:
This implementation only effect to HTML output. So we should mention users the limitation (or rename this to `html_manpages_url`).

Reply:
added a note.